fedramp-faq.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: support-page
+title: FedRAMP and GitHub
+description: GitHub Enterprise Cloud is now FedRAMP Authorized, which brings software collaboration to governments everywhere
+permalink: /fedramp-faq
+
+---
+
+
+# 🎉GitHub Enterprise Cloud + FedRAMP Tailored Authorized 🎉
+
+Governments around the world use GitHub to build software, shape policy, and share information with constituents. To ensure that governments can have access to best-in-class tools, we’ve worked with the US Federal Government to secure FedRAMP Tailored Authorization for GitHub Enterprise Cloud. 
+Government users can now rely on GitHub knowing that our platform meets the FedRAMP Tailored baseline of security standards set by our US federal government partners.
+
+With GitHub Enterprise Cloud’s FedRAMP Tailored ATO, agencies can more easily:
+
+- Securely collaborate on software development and management in the cloud
+- Modernize the way they build software
+- Take advantages of the latest features and capabilities that GitHub has to offer
+- Jump start agency authorization by reviewing our current FedRAMP Tailored ATO and authorization package
+
```

Comment:
```suggestion
<br/>
```
For some reason this butted up to the next header

Reply:
Let's do this thing.